pallets/anchors/src/benchmarking.rs

```diff
@@ -35,10 +37,24 @@ fn day<T: From<u64>>(n: u64) -> T {
 	T::from(common::MILLISECS_PER_DAY * n + 1)
 }
 
+// TODO: Move to common lib
```

Comment:
Any ideas on where to put this function? So other benchmarks may use it if needed? It can stay here for now too, until there is another reason to use it.

Reply:
Maybe `test-utils` crate is a good place for it?

Reply:
> Maybe `test-utils` crate is a good place for it?

Yes or `cfg-utils` 👍